lib/matplotlib/_mathtext.py

```diff
@@ -2901,6 +2953,30 @@ def overline(self, s, loc, toks):
         hlist = Hlist([rightside])
         return [hlist]
 
+    def overset(self, s, loc, toks):
+        assert len(toks) == 1
+        assert len(toks[0]) == 2
+        # Only one symbol
+        assert len(toks[0][1]) == 1
+        state = self.get_state()
+
+        value, symbol = toks[0]
+        symbol = symbol[0]
```

Comment:
you can just write e.g. `(value, (symbol,)), = toks` and get rid of the asserts (and likewise for `underset`.